libs/spoke-codegen/src/graphql/super-admin-people.graphql

```diff
@@ -1,6 +1,41 @@
+fragment Users on UsersList {
+  users {
+    id
+    displayName
+    email
+    isSuperadmin
+  }
+}
+
 query GetOrganizations {
   organizations {
     id
     name
   }
 }
+
+query GetSuperAdmins {
+  superadmins {
+    id
+    displayName
+    email
+    isSuperadmin
+    isSuspended
+  }
+}
+
+query GetPeople {
+  people {
+    ...Users
+  }
+}
```

Comment:
Question: Any reason for creating the Users fragment and not a Superadmins fragment for these similar queries?

Reply:
No specific reason, just been following the convention of `Data` fragment from `DataList` types. Will change this to `SuperAdmins`